charts/nri-bundle/values.yaml

```diff
@@ -1,106 +1,129 @@
-# Default values for nri-infrastructure.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# enables newrelic-infrastructure
 infrastructure:
+  # infrastructure.enabled -- Install the [`newrelic-infrastructure` chart](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure)
   enabled: true
 
-# enables nri-prometheus
 prometheus:
+  # prometheus.enabled -- Install the [`nri-prometheus` chart](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus)
   enabled: false
 
-# enables nri-metadata-injection
 webhook:
+  # webhook.enabled -- Install the [`nri-metadata-injection` chart](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection)
   enabled: true
 
-# enables kube-state-metrics
 ksm:
+  # ksm.enabled -- Install the [`kube-state-metrics` chart from the stable helm charts repository](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics)
```

Comment:
we can add it is required if infrastructure.enabled=true unless they are running their own KSM already